syntax = "proto3";
import "data_point.proto";
import "label.proto";

message DataFrame {
  repeated DataPoint data = 1;
  repeated Label label = 2;
}
